<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application cours</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="b_seconnecter">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>se connecter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_coursproposes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Les cours Proposes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_utilisateurs">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>les utilisateurs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_ajouterutilisateur">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ajouter Utilisateur</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>561</width>
      <height>301</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_proposercours">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>proposer Cours</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mescoursproposes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mes cours Proposes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enseignantcoursvalides">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mes cours valides</string>
    </property>
   </widget>
   <widget class="QLabel" name="pseudo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="type">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="quitter">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sedeconnecter">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>se deconnecter</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDescours">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
